jbpm-console-ng-executor-service/jbpm-console-ng-executor-service-api/src/main/java/org/jbpm/console/ng/es/model/RequestDetails.java

```diff
@@ -18,10 +18,12 @@
 
 import java.io.Serializable;
 import java.util.List;
-import java.util.Vector;
```

Comment:
I guess the question is why it was there in the first place :)
This class should basically be immutable as to represent the data retrieved from the Kie server, not really intended to manipulate its state.